static/js/tabpanel.js

```diff
@@ -0,0 +1,73 @@
+let timer;
+
+window.addEventListener('DOMContentLoaded', () => {
+  const tabs = document.querySelectorAll('[role="tab"]');
+
+  // Add a click event handler to each tab
+  tabs.forEach(tab => {
+    tab.addEventListener('click', function(e) {
+      const target = e.target.closest('[role="tab"]');
+      changeTabs(target);
+    });
+  });
+
+  // Select the inital active tab
+  const initalActiveTab = document.querySelector('.p-hero-tab__item[aria-selected="true"]');
+  if (initalActiveTab) {
+    playTab(initalActiveTab);
+  }
+});
+
+function changeTabs(target) {
+  const tabs = document.querySelectorAll('[role="tab"]');
+  const panels = document.querySelectorAll('[role="tabpanel"]');
+  clearInterval(timer);
+
+  // Remove all current selected tabs
+  tabs.forEach(tab => {
+    draw(0, tab.querySelector('.before'));
+    tab.setAttribute('aria-selected', false);
+  });
+
+  // Set this tab as selected
+  target.setAttribute('aria-selected', true);
+
+  // Hide all tab panels
+  panels.forEach(panel => panel.classList.remove('u-animate--reveal'));
+
+  // Show the selected panel
+  document
+    .querySelector(`#${target.getAttribute('aria-controls')}`)
+    .classList.add('u-animate--reveal');
+
+  playTab(target);
+}
+
+function playTab(tab) {
+  let start = Date.now();
```

Comment:
Is there a reason these are not declared as `const` - they don't seem to change?

Reply:
Nope, copy and paste job